bot/cogs/fun.py

```diff
@@ -39,6 +39,14 @@ def __init__(self, bot: Bot):
         if "dabato" in message.content:
             await message.add_reaction("🤔")
 
+        # React FBI OPEN UP if message contains trigger words.
+        triggers_list = ["child", "fbi", "loli", "hentai", "illegal", "maltego"]
+        react_list = ["🇫", "🇧", "🇮", "⚫", "🇴", "🇵", "🇪", "🇳", "⚪", "🇺", "🅿️"]
```

Comment:
Might be able to use the following syntax soon, if this PR is still open after #49 gets merged:
```python
react_list = self.bot.emojify('fbi open up')
```